Shoko.Server/API/v3/Models/Shoko/File.cs

```diff
@@ -105,17 +105,18 @@ public class Location
             /// <summary>
             /// The Import Folder that this file resides in 
             /// </summary>
+            [Required]
             public int ImportFolderID { get; set; }
             
             /// <summary>
             /// The relative path from the import folder's path on the server. The Filename can be easily extracted from this. Using the ImportFolder, you can get the full server path of the file or map it if the client has remote access to the filesystem. 
             /// </summary>
+            [Required]
             public string RelativePath { get; set; }
             
             /// <summary>
             /// Can the server access the file right now
             /// </summary>
-            [JsonRequired]
```

Comment:
Unrelated change?

Reply:
No. It's for submitting file locations when relocating files. Also with the new default of including every field value by default then I don't see why this attribute is still required as it's always sent from the server-side and can be omitted when used in a body sent from the client-side (since it have no effect on the server).